apertium_apy/handlers/per_word.py

```diff
@@ -135,13 +136,11 @@ def handle_output(output):
         result = pool.apply_async(process_per_word, (self.analyzers, self.taggers, lang, modes, query))
```

Comment:
Did we leave `PerWordHandler.get` as a coroutine?

Reply:
Thanks, fixed in 92a477b357662cf8f9ca9251b845826a0680c5f7